com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -337,6 +353,44 @@ void DrawLightingSettings()
             EditorGUILayout.EndFoldoutHeaderGroup();
         }
 
+        void DrawShadowResolutionTierSettings()
+        {
+            // UI code adapted from HDRP U.I logic implemented in com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/SerializedScalableSetting.cs )
+
+            var rect = GUILayoutUtility.GetRect(0, float.Epsilon, EditorGUIUtility.singleLineHeight, EditorGUIUtility.singleLineHeight);
+            var contentRect = EditorGUI.PrefixLabel(rect, Styles.additionalLightsShadowResolutionTiers);
+
+            EditorGUI.BeginChangeCheck();
+
+            const int k_ShadowResolutionTiersCount = 3;
```

Comment:
I think this will trigger error in formatter, IIRC we only use prefix k_ const (My suggestion to change name)?

Reply:
Thanks, it seems that the formatter job did not have problem with it:
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/universal%252Fcustom-shadow-resolution/.yamato%252F_formatting.yml%2523formatting/4550981/job/(log:Execution)

I think I also found other occurrences of the same naming pattern in the project (UniversalRenderPipelineAsset.k_ShadowCascadeMinCount).